.idea/codeStyles/Project.xml

```diff
@@ -6,15 +6,6 @@
           <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
         </value>
       </option>
-      <option name="PACKAGES_IMPORT_LAYOUT">
```

Comment:
Is this change necessary ?

Reply:
it is something that my AS always asks for ...it seems that it slept here...I would actually like to commit it if it doesn't bother anyone and avoid removing it all the time manually from the commit.

Reply:
Or we could remove the `codestyle/*` from tracked files altogether. We already have ktlint to ensure we share the same formatting